### DISCUSSION TITLE
3D MAX에서 작업한 맵핑이 유니티에서 적용이 안 됩니다.

### DISCUSSION THREAD

**Discussion by hsdstudent** [naverz/zepeto-studio-kor#6] [5. Q&A]
맥스에서 모델링과 맵핑 작업을 끝내고 유니티로 옮겼는데, 모델링은 정상적으로 보이는데 맵핑 작업한 게 보이지 않습니다. max 파일, fbx 파일 둘 다 해봤고 file - export - export 클릭한 다음 뜨는 설정창에서 'Embed Media'도 체크 했는데 적용이 안 되고 있습니다. 혹시 해결 방법을 알고 계실까요?

**Comment by pbatlhazer** [comment]
유니티에서 맥스로 export한 fbx가 깨지는 문제 같아요. 유니티 가이드를 확인해 보시는게 좋을것 같아요. 

- 3D Studio Max에서 오브젝트 임포트
https://docs.unity3d.com/kr/530/Manual/HOWTO-ImportObjectMax.html
https://magatron.tistory.com/93
